config/locales/en.yml

```diff
@@ -1531,6 +1531,7 @@ en:
     contrast: Mastodon (High contrast)
     default: Mastodon (Dark)
     mastodon-light: Mastodon (Light)
+    system: System settings
```

Comment:
‘Sync with system’ or ‘System theme’ probably would be better.

Reply:
Or maybe something like “Automatic”

Reply:
Thanks both for the input! I'll change this to "Automatic (use system theme)" for English.